### PR TITLE
Week 10

### PR DESCRIPTION
**Pull request by haroldtreen**
Wrote what is currently here in about an hour last night. Need to rework it, but would love feedbackkk ❤️

### CODE REVIEW

**Comment by haroldtreen** [comment]
Andd pinging my fave blog reviewer @vaibhavsagar

**Comment by qwo** [comment]
Good Read 👍 @haroldtreen .

I have the same issue with tracking knowledge that I see and immediacy. 
I save lots of ideas and cool things I see but I don't think honestly think I will be able to reliably recall it when I need it. I've been toying about how to aggregate articles I care about without me manually curating them. I'm going to check out https://epub.press !

**Comment by haroldtreen** [comment]
Yay! Thanks for the feedback @stanzheng. 

I've definitely found EpubPress has helped me curate content.
I'm undecided whether putting stuff aside and then searching in that aside is more efficient then just searching google anytime I have a problem... google tends to answer my questions pretty quick. 

These are the questions :P

File: _posts/2016-12-06-recurse-center-week-10.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Week[9] @ The Recurse Center 🔎"
+comments: true
+categories: tech recurse
+---
+
+### Overall
+
+I'm currently trying to distill through what is essential, and focus on that. Stripping away the excess has left more energy for the important pursuits.
```
Comment (vaibhavsagar) [inline _posts/2016-12-06-recurse-center-week-10.md]: How does this sentence sound instead: _I'm currently trying to distill what is essential, and focusing on that_?

File: _posts/2016-12-06-recurse-center-week-10.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+### Emotional 😰
+
+### **Frantic**
+
+Feels have been evolving recently, but at the moment I feel frantic. There's now only **2** weeks left.
+
+I was reflecting on the first half versus the second half. It seems like I got more done in the first half...
```
Comment (vaibhavsagar) [inline _posts/2016-12-06-recurse-center-week-10.md]: I know exactly how this feels. I certainly felt like I was coasting on the momentum I had in my first half, but then I was describing weeks 7-9 to an alum and to him it sounded like I had accomplished a lot. Try doing the same thing and you might be pleasantly surprised at all you've done :smile:. Remember also that I devoted my entire batch to getting better at Haskell after spending a couple of months studying it on my own so from my perspective you've made a ton of progress in a few weeks.

File: _posts/2016-12-06-recurse-center-week-10.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+I'm feel frantic to find something I can build in two weeks that will prove that **I am better**. A **DRASTICALLY** better programmer.
+
+This desire has also motivated me to constantly be working.
+
+> It's 9pm... but I don't feel accomplished. Gah! Stay later, work longer, we'll be drastically better soon.
+
+Maybe if I went back 3 months, the improvement would be more obvious. I feel drastically better than a year ago, but it's hard to know how much of that has come from RC.
+
+Anyway... I'm frantic to feel accomplished, and it hasn't hit me yet.
```
Comment (vaibhavsagar) [inline _posts/2016-12-06-recurse-center-week-10.md]: One time where I recently felt that I'd drastically improved as a programmer was when I revisited blog posts that had previously gone completely over my head and I was able to comprehend them.

Remember also that you'll still be able to build things after your batch is over. In fact, some of my most productive times during my batch were when I wasn't even at RC. It is easier for me to sit down and write code when I'm not distracted by the opportunity to hang out with 70 other lovely people :smile:

In short: I wouldn't worry too much about being productive.

File: _posts/2016-12-06-recurse-center-week-10.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+#### **Blob**
+
+Days on end of sitting at a computer is really starting to make me feel out of shape. REALLY looking forward to settling in somewhere and being able to do a better job of focusing on my health.
+
+### </End>
+
+Woo! We did it - and it's not even Friday! Getting back on track 😊.
+
+While I may not be feeling like a massively better programmer at the moment, I sure feel like a much better person. I was thinking about how much time I've invested in blogging. I think it's helped maximize what I get out of all these discussions and questions. So that's been nice!
```
Comment (vaibhavsagar) [inline _posts/2016-12-06-recurse-center-week-10.md]: This echoes what @leotrs said when I met him for the first time: "being at RC made me not just a better programmer, but a better person". I'm glad you feel the same way :smile:

File: _posts/2016-12-06-recurse-center-week-10.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+Thanks for reading ❤️.
+
+*Hearts to @vaibhavsagar who I am going to mention before even getting any feedback from, because I have much trust in him!*
```
Comment (vaibhavsagar) [inline _posts/2016-12-06-recurse-center-week-10.md]: :revolving_hearts:

File: _posts/2016-12-06-recurse-center-week-10.md (hunk 1 above)
Comment (haroldtreen) [reply _posts/2016-12-06-recurse-center-week-10.md]: ...Maybe more englishy? I'll update it and see if I feel otherwise before I'm done with other edits.

File: _posts/2016-12-06-recurse-center-week-10.md (hunk 2 above)
Comment (haroldtreen) [reply _posts/2016-12-06-recurse-center-week-10.md]: Today at feelings checkin, others felt like the first half had been better too.

There's still 2 weeks and I think I can do a bunch in the remaining time. But first half I was working on project related stuff... and I suppose that was more gratifying.

File: _posts/2016-12-06-recurse-center-week-10.md (hunk 3 above)
Comment (haroldtreen) [reply _posts/2016-12-06-recurse-center-week-10.md]: Defs super optimistic about all the growth to come after batch. I've also found weekends to be super productive. 

RC keeps talking about "Becoming a better programmer" so it's hard not to use that as a metric of success. Damn you branding!

File: _posts/2016-12-06-recurse-center-week-10.md (hunk 4 above)
Comment (haroldtreen) [reply _posts/2016-12-06-recurse-center-week-10.md]: Harold's version: "I'm not a better programmer, but I'm a better person!" 

1/2 isn't bad 😛

File: _posts/2016-12-06-recurse-center-week-10.md (hunk 5 above)
Comment (haroldtreen) [reply _posts/2016-12-06-recurse-center-week-10.md]: 😄

**Comment by haroldtreen** [comment]
Thanks again for the words of encouragement @stanzheng & @vaibhavsagar .

Much hearts to you! ❤️ 

LGTM! 🚀
